VirtoCommerce.Platform.Web/Controllers/HomeController.cs

```diff
@@ -68,5 +80,17 @@ private static License LoadLicense()
 
             return license;
         }
+
+        private string GetFavIcon()
+        {
+            string result = null;
+            var uiSettings = _settingsManager.GetSettingByName("VirtoCommerce.Platform.UI.Customization");
+            if (uiSettings != null)
+            {
+                var jObject = JObject.Parse(uiSettings.Value);
+                result = (string)jObject.SelectToken("favicon");
```

Comment:
Need to take into account what there is may be platform installations  with already exist stored settings which may not   contain the new  `favicon` property.   So then this lines mustn't cause an exception.

Reply:
Handled possible JObject.Parse exceptions and made explicit expression suppress on jObject.SelectToken.